src/main/java/fr/insee/genesis/configuration/Config.java

```diff
@@ -19,7 +21,15 @@ public class Config {
 	@Value("${fr.insee.genesis.sourcefolder.specifications}")
 	private String specFolderSource;
 
-	@Value("${fr.insee.genesis.logfolder}")
-	private String logFolder;
-
+	private final String logFolder;
+
+	//Extract log folder from log filename property
+	public Config(@Value("${fr.insee.properties.log.filename}") String logFileName) {
+		Path logFileNamePath = Path.of(logFileName);
+		if(logFileNamePath.getFileName().toString().contains(".")){
```

Comment:
Is it to test if it's a file ?
boolean isDirectory = Files.isDirectory(file);   // Check if it's a directory

Reply:
Files.isDirectory returns false if directory does not exists in file system, here we just want to be sure that logFolder is a folder even if the folder does not exists